src/components/RenderRouter/BlogReader.tsx

```diff
@@ -71,19 +77,48 @@ export default class VideoPlayer extends React.Component<Props, State> {
     )
   }
 
-  render() {
+  downloadButton(pdf_url: string) {
     return (
-      (this.state.data !== null) ?
-        <div className="blog">
-          <div className="link-to-main-blog-page" onClick={()=>this.navigateUrl("/blog")}>Blog<img className="dropdown-caret" src="/static/svg/Dropdown Caret.svg" alt=""></img></div>
-            <div className="blog-content">
-              <h1 className="blog-h1" >{this.state.data.blogTitle}</h1>
-              <div className="blog-details">by <span className="blog-author">{this.state.data.author}</span> on {this.state.data.publishedDate}</div>
-              <div className="ShareButtonDesktop">{this.shareButton()}</div>
-              <div className="blog-body">{ReactHtmlParser(this.state.data.content)}</div>
-              <div className="ShareButtonMobile">{this.shareButton()}</div>
-            </div>
-        </div> : null
+      <Button className="download-custom" onClick={()=>this.navigateNewUrl(pdf_url)}><img className="button-icon" src="/static/svg/Download-White.svg" alt=""/>PDF</Button>
     )
   }
+
+  render() {
+    if (this.state.content.style === "blog") {
+      return (
+        (this.state.data !== null) ?
```

Comment:
You can move this to the beginning of the function to remove the ternary operator here and in the other branch.